pixart.js

```diff
@@ -1 +1,13 @@
-
+$(document).ready(function(){
+  $("#set-color").on("click", function(e){
+    e.preventDefault();
+    console.log("test!");
+    $(".brush").css("background-color", $("#color-field").val());
+  });
+    for(var i = 0; i < 8000; i++){
+      $("body").append("<div class = 'square'></div>");
+    }
+    $(".square").on("mouseover", function(){
+      $(this).css("background-color", $("#color-field").val());
```

Comment:
This works, but is not quite as robust. If I 'set' a color (by clicking the set button), then clear the input field, this code won't use the color that I set, as it is still pulling from the input field.

Instead, I'd do: 
```js
var color = $(".brush").css("background-color");
```
which gets the value that was 'set'.